### PR TITLE
use yyyy.mm.dd format for date input if the date display format is ISO

### LINKED ISSUE #891: yyyy-mm-dd format for inputting a date
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

I usually use yyyy/mm/dd as a date format, so inputting a date in dd.mm.yyyy or mm.dd.yyyy format is a bit of pain for me.

### Suggested solution

When I choose 'yyyy-mm-dd' as the date display format, it's perfect if the input format also changes to 'yyyy-mm-dd'

### Alternative

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by syobocat**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Related Tickets & Documents

Closes #891 

### Mobile & Desktop Screenshots/Recordings

Before:
スクリーンショット 2024-08-13 21 49 07

After:
スクリーンショット 2024-08-13 21 49 45

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by syobocat** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/ts/component/block/dataview/cell/text.tsx [hunk 1]
```diff
@@ -75,14 +75,26 @@ const CellText = observer(class CellText extends React.Component<I.Cell, State>
...
-					ph.push('mm.dd.yyyy');
-				} else {
-					ph.push('dd.mm.yyyy');
-				};
+				switch (viewRelation.dateFormat) {
+					case I.DateFormat.ISO: {
+						mask.push('9999.99.99');
+						ph.push('yyyy.mm.dd');
+						break;
+					}
```
Comment (ra3orblade) [inline src/ts/component/block/dataview/cell/text.tsx:86]: Fix formatting, add missing semicolons everywhere.

**Review by ra3orblade — CHANGES_REQUESTED** [review]

File: src/ts/component/block/dataview/cell/text.tsx (hunk 1 above)
Comment (ra3orblade) [reply src/ts/component/block/dataview/cell/text.tsx:86]: Check PR changes carefully please, not fixed.

Comment (syobocat) [reply src/ts/component/block/dataview/cell/text.tsx]: Oh I'm sorry, I didn't know that `switch` statements should have a semicolon. This should be OK now.

**Review by ra3orblade — APPROVED** [review]

**Pull request merged**